src/Connect.js

```diff
@@ -111,16 +103,28 @@ class Uport {
     return this.txObjectHandler(txobj, uriHandler)
   }
 
+  addAppParameters (txObject, callbackUrl) {
```

Comment:
@zachferland It maybe the functional programmer in me, because the txObject that gets passed in are the basic parameters, so I'm just adding the App and not transactions specific parameters here. I'm open.